src/Access/SettingsProfilesCache.cpp

```diff
@@ -237,5 +246,18 @@ std::shared_ptr<const SettingsChanges> SettingsProfilesCache::getProfileSettings
     return res;
 }
 
+Strings SettingsProfilesCache::getProfilesNames(const std::vector<UUID> & profiles) const
```

Comment:
It seems there is no point for this function to handle multiple profiles at once. This logic can be implemented in the caller and so it would allow make this function more simple. I mean this function can be declared instead as `std::optional<String> getProfileName(const UUID & profile_id) const` and the caller will iterate through IDs if it wants.

Reply:
Ok, fixed